integration/apps/rails-five/app/controllers/basic_controller.rb

```diff
@@ -0,0 +1,36 @@
+require "#{Rails.root}/app/jobs/test_job"
+
+class BasicController < ApplicationController
+  # Reads & writes from cache, reads from DB, and queues a Resque job.
+  #
+  # Example trace:
+  #
+  # ----------------------- Rack -----------------------------
+  #    -------------- ActionController --------------------
+  #      --- ActiveSupport -- ActiveRecord -- Resque ---
+  #        ----- Redis ----                 -- Redis --
+  #
+  def default
+    # Read from the database
+    records = Test.where(version: 0)
+
+    # Queue job
+    Resque.enqueue(Jobs::Test, job_id: request.request_id, records: records.map(&:to_json))
+
+    # Return response
+    render json: { job_id: request.request_id }
+  end
+
+  # Runs a recursive implementation of fibonacci.
+  # Provides a basic load on CPU, stack frames, response time.
+  def fibonacci
+    fib(rand(25..35))
+    head :ok
+  end
```

Comment:
Minor: maybe return the value, like in the rack variant, just have see some output?